source/developers/concepts/hooks_extension_framework.rst

```diff
@@ -163,6 +163,37 @@ Filters are useful when you need an immediate response that directly modifies th
 
 We encourage you to review the list of use cases for events and filters to draw inspiration from real-life scenarios and see if your use case aligns with any of them. Also, maybe your feature can be implementing using the framework but there's not an available event or filter for it yet. In that case, consider proposing a new event or filter to the community!
 
+What's Next?
```

Comment:
I'm still thinking this is missing something. I'll re-review it tomorrow so I can open it for others to review.

Reply:
I think the high-level overview of the contribution process to the framework is good as it is.